index.html

```diff
@@ -16,8 +16,19 @@
       },
       {
         name: "Yuri Shkuro",
-        company: "Facebook",
+        company: "Meta",
         w3cid: "104074"
+      },
+      {
```

Comment:
since we are getting closer with this spec, we may want to start collecting Acknowledgements 

I used this method for trace-context: https://github.com/w3c/trace-context/issues/268

Reply:
Good idea. Maybe we should do that in a separate PR though? Last time I think you asked each person for a confirmation that they are okay with being mentioned in the acknowledgements section.

Here is the output of `git shortlog --summary --numbered --email`, maybe we want to use that as a basis?

```
  49  Daniel Khan <daniel@khan.io>
    48  Sergey Kanzhelev <S.Kanzhelev@live.com>
    39  Daniel Dyla <dyladan@users.noreply.github.com>
    30  J. Kalyana Sundaram <kalyanaj@microsoft.com>
    21  Sergey Kanzhelev <SergKanz@microsoft.com>
    14  Alois Reitbauer <alois@reitbauer.cc>
    13  Reiley Yang <reyang@microsoft.com>
    11  Armin Ruech <armin.ruech@dynatrace.com>
     7  plehegar <plh@w3.org>
     5  Morgan McLean <morgan_mclean@hotmail.com>
     4  Adrian Cole <adriancole@users.noreply.github.com>
     4  Anuraag Agrawal <anuraaga@gmail.com>
     4  Bogdan Drutu <bdrutu@google.com>
     4  Philippe Le Hegaret <plh@w3.org>
     4  Wendy Seltzer <wendy@seltzer.org>
     3  Bogdan Drutu <bogdandrutu@gmail.com>
     3  Matthew Wear <matthew.wear@gmail.com>
     3  Yuri Shkuro <github@ysh.us>
     3  吴晟 Wu Sheng <wu.sheng@foxmail.com>
     2  Sergey Kanzhelev <sergkanz@microsoft.com>
     2  Sergey Kanzhelev <skanzhelev@google.com>
     1  Adrian Cole <acole@pivotal.io>
     1  Bastian Krol <bastian.krol@ibm.com>
     1  Ben Sigelman <bhs@users.noreply.github.com>
     1  Christian Weiss <christian@chwe.at>
     1  Christoph Neumüller <c.neumueller@gmail.com>
     1  Dominique Hazael-Massieux <dom@w3.org>
     1  Erika Arnold <erabug@users.noreply.github.com>
     1  Felix Barnsteiner <felixbarny@users.noreply.github.com>
     1  Fuqiao Xue <xfq@w3.org>
     1  Heinz N. Gies <heinz@licenser.net>
     1  Ivan Kirichenko <iddqd07@yandex.ru>
     1  John Lian <jlian@users.noreply.github.com>
     1  Matthew Grossman <matt@mrgrossman.com>
     1  Nik Molnar <nikmd23@gmail.com>
     1  Pavol Loffay <p.loffay@gmail.com>
     1  Samuel Weiler <weiler@w3.org>
     1  Sergey Kanzhelev <9950081+SergeyKanzhelev@users.noreply.github.com>
     1  Tom Sugden <tom.sugden@gmail.com>
     1  Victor Soares <vsoares@newrelic.com>
     1  caribouW3 <carine@w3.org>
     1  rektide <rektide@voodoowarez.com>
```

Reply:
this is great, would be nice to create an issue, mention people and add people who will reply back. I also feel like the cutline of 5 may be ok and below the should be review of non-code contributions.

Reply:
I've created https://github.com/w3c/baggage/issues/122 for this.